tests/test_http.py

```
@@ -54,6 +55,102 @@ def test_auto_resume_garth_token(
+
+def test_auto_save_on_resume_login(monkeypatch: pytest.MonkeyPatch):
+    with tempfile.TemporaryDirectory() as tempdir:
+        monkeypatch.setenv("GARTH_HOME", tempdir)
+        monkeypatch.delenv("GARTH_TOKEN", raising=False)
+
+        client = Client()
+
+        import garth.sso
+
+        mock_oauth1 = OAuth1Token(
+            oauth_token="test_token",
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to argument: "oauth_token"

(S106)